WalletWasabi.Gui/Tabs/WalletManager/LoadWallets/LoadWalletViewModel.cs

```diff
@@ -117,14 +120,15 @@ public WalletViewModelBase SelectedWallet
 		}
 
 		public SourceList<WalletViewModelBase> RootList { get; private set; }
-		public ReactiveCommand<Unit, Unit> LoadCommand { get; }
+		public ReactiveCommand<Unit, IDisposable> LoadCommand { get; }
```

Comment:
Why did we change the return type?

Reply:
I changed the LoadCommand property's type in order to use expression body (otherwise you get a conversion error)

Before:
```
LoadCommand = ReactiveCommand.Create(()=>
{
	RxApp.MainThreadScheduler.Schedule(async () =>
	{
		await LoadWalletAsync();
	});
}, this.WhenAnyValue(x => x.SelectedWallet, x => x?.WalletState).Select(x => x == WalletState.Uninitialized));
```
This PR:
```
LoadCommand = ReactiveCommand.Create(() =>
	RxApp.MainThreadScheduler.Schedule(async () => await LoadWalletAsync())
		.DisposeWith(Disposables),
	this.WhenAnyValue(x => x.SelectedWallet, x => x?.WalletState)
		.Select(x => x == WalletState.Uninitialized));
```

Reply:
Why not add.Select(_=> unit.default) and leave return type as is.

Reply:
You mean like this?
![Capture](https://user-images.githubusercontent.com/52379387/79287532-5bf9aa80-7ec4-11ea-84e5-8fc7038fc624.PNG)

Reply:
This is good like this IMO.